docs/3.0.0-beta.x/plugins/documentation.md

```diff
@@ -171,6 +176,58 @@ To access your documentation on a custom path, you will have to update the `path
 }
 ```
 
+### Indicate which plugins documentation to generate
+
+To generate a plugin documentation, you will need to indicate the list of all the plugins for which to you wish to generate documentation. In order to do that you need to update the `pluginsForWhichToGenerateDoc` key. Leaving this key with an empty array `[]` means that not any plugin documentation will be generated.
+
```

Comment:
Nice, can you please add that when the key `pluginsForWhichToGenerateDoc` is not specified in the settings, then the documentation will be generated for all the plugins ?

Reply:
@petersg83 this means that is should not be added in the default settings file. For now, it is part of the default settings file so that people will always generate the default plugins documentation. If the variable is not present then it takes the default one as it is not overwritten and if you leave it empty then it overwrites the default one and does not generate any doc for the plugins. Do you then want to provide a default setting file without pluginsForWhichToGenerateDoc key and check in the code if this key is not present then generate the default set of plugins?

Reply:
yes exactly :)
Removing it from the default setting file in order to generate the doc for all plugins by default. If the user wants to select which plugins he wants, thenn, by reading the doc, he will add the key in his own settings.

You only need to remove the key from the default config and it works, no need to change other parts of the code normally

Reply:
@petersg83 updated the PR. Thanks for your feedback. @alexandrebodin are you ok with the latest updates?